### PR TITLE
Set up list structure

### PR DESCRIPTION
**Pull request by ShadyShawkat**
# Features

- [x] Use Web packs.
- [x] Add list HTML structure and style.
- [x] Load static TODO-list in the page dynamically.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+window.onload = () => {
+  const todoList = document.querySelector('.todo-list');
+  todoArr.forEach((todoItem) => {
+    const li = document.createElement('li');
+    li.className = 'todo-item';
+    li.innerHTML = `
+    <label class="${todoItem.completed ? 'todo-completed' : ''}">
+      <input type="checkbox" class="todo-item-check" ${todoItem.completed ? 'checked' : ''}>
+      ${todoItem.describtion}
```
Comment (Dipeshtwis) [inline src/index.js]: [OPTIONAL]

- There should be `description` instead of `describtion`.

File: README.md [hunk 2]
```diff
@@ -1,59 +1,36 @@
...
-### Usage
-
-### Run tests
-
-### Deployment
-
-
+1- Navigate to the button on the top right green "code" button.
+2- In the drop down menu choose "Download Zip".
+3- After extracting the files, you'll have my project on your local machine.
```
Comment (Dipeshtwis) [inline README.md]: - [ ] I would recommend writing all the clear steps and commands which is necessary to run the project locally. Like `npm i`, `npm start`.

**Review by Dipeshtwis — CHANGES_REQUESTED** [review]
Hi @ShadyShawkat, 

:+1: This is good to make it this far, your project is really nice, congratulations on the good work you have done so far.
:+1: The project setup is nice.

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


 ## Required Changes ♻️

**_Check the comments under the review_**

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

**_Check the comments under the review_**

Cheers and Happy coding!👏👏👏
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**_Dipesh Kumar_**


------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project(check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Emmyn5600 — APPROVED** [review]
STATUS: APPROVED :+1: 

Hello, team @ShadyShawkat   👋🏾 👋🏾 congratulations on making it so far on this milestone.

## Optional suggestions
- N/A

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

**Pull request merged**